packages/turf-helpers/index.js

```diff
@@ -248,6 +248,7 @@ function multiPolygon(coordinates, properties) {
  */
 function geometryCollection(geometries, properties) {
     if (!geometries) throw new Error('geometries is required');
```

Comment:
We can add those in the tests, but `!geometries` would not exclude empty geometries inside, it would only exclude `geometries: null` or `undefined.

lets add them to the tests and see what happens from there :) lol